erpnext/accounts/page/pos/pos.js

```diff
@@ -1037,7 +1039,7 @@ erpnext.pos.PointOfSale = erpnext.taxes_and_totals.extend({
 		this.numeric_keypad.show();
 	},
 
-	make_item_list: function () {
+	make_item_list: function (customer=null) {
```

Comment:
no need for `=null`: it is `undefined` (falsy) by default

Reply:
fixed